vision/google/cloud/vision/entity.py

```diff
@@ -63,13 +63,33 @@ def from_api_repr(cls, response):
         bounds = Bounds.from_api_repr(response.get('boundingPoly'))
         description = response['description']
         locale = response.get('locale', None)
-        locations = [LocationInformation.from_api_repr(location)
+        locations = [LocationInformation.from_api_repr(location['latLng'])
                      for location in response.get('locations', [])]
```

Comment:
I realize it's not in this PR, but it's good form to use `()` as your default empty container (since it takes up less space than `[]` and is just being thrown away anyhow)